db/init_db.js

```diff
@@ -57,206 +57,209 @@ async function buildTables() {
         imageUrl: "",
         price: 8.24,
         inStock: true,
         category: "cat",
-      },
-      {
+      }),
+      createProduct({
         name: "Non-Clumping Cat Litter",
         description: "Febreze scented non-clumping clay cat little, 35-lb bag",
         imageUrl: "",
         price: 13.99,
         inStock: true,
         category: "cat",
-      },
-      {
+      }),
+      createProduct({
         name: "Ultra Rubber Ball",
         description: "Ultra rubber ball tough dog toy, Medium, 2 pack",
         imageUrl: "",
         price: 5.95,
         inStock: true,
         category: "dog",
-      },
-      {
+      }),
+      createProduct({
         name: "Life Protection Dry Dog Food",
         description:
           "Life protection formula adult chicken & brown rice recipe dry dog food, 30-lb bag",
         imageUrl: "",
         price: 51.98,
         inStock: true,
         category: "dog",
-      },
-      {
+      }),
+      createProduct({
         name: "Aquarium Starter Kit",
         description: "Tropical panaview aquarium starter kit, 5-gal",
         imageUrl: "",
         price: 39.99,
         inStock: true,
         category: "fish",
-      },
-      {
+      }),
+      createProduct({
         name: "NeoGlow Aquarium Starter Kit",
         description: "LED neoGlow aquarium starter kit, Pink, 5.5-gal",
         imageUrl: "",
         price: 19.97,
         inStock: true,
         category: "fish",
-      },
-      {
+      }),
+      createProduct({
         name: "Shale Ledge",
         description: "Shale ledge aquarium & terrarium decor, 1-count",
         imageUrl: "",
         price: 13.21,
         inStock: false,
         category: "reptile",
-      },
-      {
+      }),
+      createProduct({
         name: "Rock Food & Water Dishes",
         description: "Repti rock reptile rock food & water dishes, Small",
         imageUrl: "",
         price: 5.49,
         inStock: true,
         category: "reptile",
-      },
-      {
+      }),
+      createProduct({
         name: "Small Animal Cage",
         description:
           "Home sweet home plastic small animal cage, Color Varies, 28-in",
         imageUrl: "",
         price: 26.99,
         inStock: true,
         category: "small animal",
-      },
-      {
+      }),
+      createProduct({
         name: "Feeder & Water Fountain",
         description: "Mini rabbit feeder & water fountain, 48-oz",
         imageUrl: "",
         price: 7.99,
         inStock: true,
         category: "small animal",
-      },
-      {
+      }),
+      createProduct({
         name: "Orthopedic Sofa Dog Bed",
         description: "Plush & suede orthopedic sofa dog bed, Gray, Large",
         imageUrl: "",
         price: 38.99,
         inStock: false,
         category: "dog",
-      },
-    ];
+      });
 
     const products = await Promise.all(initialProducts.map(createProduct));
```

Comment:
It looks like you got rid of initialProducts and called the function for each product instead? This won't work in that case.

Reply:
hmmmm... i had originally done the maps because I was going to do the order files the same way but when I was working on that file earlier this week I figured out a way to do it without using maps so I removed maps from the init.db file. Somehow they were added back in when I merged with dev in class on wednesday. Would you be okay with me removing maps from this file entirely?